### PR TITLE
Updates physikon 2022

### PR DESCRIPTION
**Pull request by stefangri**

### CODE REVIEW

**Comment by jpherdi** [comment]
The build fails. You need to fix that first.

**Comment by stefangri** [comment]
Website runs locally on my PC. Does it work for you @jpherdi ?

**Review by donnama — APPROVED** [review]

File: _includes/exhibitors.html [hunk 1]
```diff
@@ -1,56 +1,75 @@
...
   <h5>Ausstellergebühr</h5>
   <p style="font-size: 1.2rem">
-    Der Höhe des Ausstellerbeitrags für die PhysiKon 2021 können Sie selbst bestimmen.
-    Selbstverständich bekommen Sie über Ihren Beitrag eine Rechnung.
-    Alternativ können Sie auch an unseren gemeinnützigen Verein PeP et al. e.V. spenden und bekommen eine Spendenquittung.
-    Mit dem Geld, das wir mit der PhysiKon 2021 einnehmen, finanzieren wir Projekte und Stipendien für die Physikstudierenden der TU Dortmund.
-    Falls Sie sich über unsere Arbeit informieren möchten, besuchen Sie gerne die <a href="https://pep-dortmund.org/" target="_blank" rel="noopener noreferrer">Webseite von PeP et al.</a>
+    Der Ausstellerbeitrag für die PhysiKon 2022 beträgt 1000€.
+    Selbstverständich stellen wir Ihnen eine Rechnung über den Beitrag aus.
+    Zudem freut sich der gemeinnützige Verein PeP et al. e.V. über Spenden mit denen
```
Comment (maxnoe) [inline _includes/exhibitors.html:69]: , mit denen

File: _includes/exhibitors.html [hunk 2]
```diff
@@ -1,56 +1,75 @@
...
   <p style="font-size: 1.2rem">
-    Der Höhe des Ausstellerbeitrags für die PhysiKon 2021 können Sie selbst bestimmen.
-    Selbstverständich bekommen Sie über Ihren Beitrag eine Rechnung.
-    Alternativ können Sie auch an unseren gemeinnützigen Verein PeP et al. e.V. spenden und bekommen eine Spendenquittung.
-    Mit dem Geld, das wir mit der PhysiKon 2021 einnehmen, finanzieren wir Projekte und Stipendien für die Physikstudierenden der TU Dortmund.
-    Falls Sie sich über unsere Arbeit informieren möchten, besuchen Sie gerne die <a href="https://pep-dortmund.org/" target="_blank" rel="noopener noreferrer">Webseite von PeP et al.</a>
+    Der Ausstellerbeitrag für die PhysiKon 2022 beträgt 1000€.
+    Selbstverständich stellen wir Ihnen eine Rechnung über den Beitrag aus.
+    Zudem freut sich der gemeinnützige Verein PeP et al. e.V. über Spenden mit denen 
+    Projekte und Stipendien für die Physikstudierende der TU Dortmund finanziert werden.
```
Comment (maxnoe) [inline _includes/exhibitors.html:70]: Physikstudierende*n*

**Pull request merged**